packages/rke2-traefik-crd/package.yaml

```diff
@@ -0,0 +1,4 @@
+url: https://traefik.github.io/charts/traefik-crds/traefik-crds-1.11.0.tgz
```

Comment:
I am concerned that we're going to have to patch the chart version or something here, otherwise helm will think that the chart is being downgraded from 34.2.002 -> 1.11.000

Maybe that's fine and it won't actually care, but we should test.

Reply:
Patched the chart name, hopefully the version works out.